lib/dev/frame-script.js

```diff
@@ -82,21 +85,21 @@ const observer = {
       observerService.removeObserver(observer, topic);
     }
     else if (document === content.document) {
-      if (topic === "content-document-interactive") {
+      if (topic.endsWith("-document-interactive")) {
```

Comment:
> this is an unrelated change, please separate this change in to another bug.

It is related because if content is chrome privileged topic will be `chrome-document-interactive"